packages/@okta/vuepress-site/guides/sign-into-mobile-app/stay-signed-in/ios/refresh.md

```diff
@@ -0,0 +1,12 @@
+Use the `renew` method to get a new access token:
```

Comment:
@IldarAbdullin-okta  How will the developer know they need to do this? Wait for a 401 response from their API, or proactively check?

Reply:
Will add example